Assets/GothicVR/Scripts/Caches/AssetCache.cs

```diff
@@ -161,7 +161,17 @@ public PxSoundData<float> TryGetSound(string key)
             if (soundCache.TryGetValue(preparedKey, out PxSoundData<float> data))
                 return data;
             
-            var wavFile = PxSound.GetSoundArrayFromVDF<float>(PhoenixBridge.VdfsPtr, $"{GetPreparedKey(key)}.wav");
+            PxSoundData<float> wavFile = new PxSoundData<float>();
+
+            try
+            {
+                wavFile = PxSound.GetSoundArrayFromVDF<float>(PhoenixBridge.VdfsPtr, $"{GetPreparedKey(key)}.wav");
+            }
+            catch
+            {
+                Debug.LogError("AccessViolationException: Sound not found.");
+                return null;
```

Comment:
Ah! One more thing: Please don't return null! Please set the cache key. Otherwise it will try to load it again and again.